### ISSUE TITLE
Make the highlights a bit lighter in dark mode

### ISSUE THREAD

**Issue by saturnonearth** [antfu/vscode-theme-vitesse#28]
### Clear and concise description of the problem

I love the theme, but wish the selections of item were a bit brighter so I could tell I was actually selecting an item.

### Suggested solution

Make the highlight or selection a bit brighter in dark mode.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by bufan-huang** [comment]
1： Under the dark theme, highlight more prominently when selected
[[](url)](https://github.com/antfu/vscode-theme-vitesse/pull/29)

**Comment by aloniacode** [comment]
In dark mode I could barely see the background color I was selecting, which made it hard to know what I was choosing.

**Comment by dfeeney39** [comment]
I agree, I love this theme, but it is really tough to see what file is currently selected in the navigation bar. 

And thanks for creating this! I really enjoy it.

**Comment by tofrankie** [comment]
I also think selection background is a bit weak, but we can customize the color to override Vitesse's default color. 

For example:

```json
{
  "workbench.colorTheme": "Vitesse Dark",
  "workbench.colorCustomizations": {
    "[Vitesse Dark]": {
      "editor.selectionBackground": "#4774f666",
      "editor.selectionHighlightBackground": "#4774f633",
      "list.hoverBackground": "#4773f615",
      "list.activeSelectionBackground": "#4774f666",
      "list.inactiveSelectionBackground": "#4774f631",
      "quickInputList.focusBackground": "#4773f615",
      "terminal.selectionBackground": "#4774f6b3"
    }
  }
}
```
